docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,188 @@
+krijgen in de wijze van (functioneel) documenteren van API’s, als onderdeel van
+koppelvlakdocumentatie]
+
+| **Aspect**      | **Beschrijving**                                                                                                                                                                                  |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Doel            | Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van zaaktypecatalogus (ZTC) inclusief daarinopgenomen zaaktypen, statustypen, roltypen en informatieobjecttypen, rollen en relaties naar betrokkenen, objecten, zaaktypen en statustypen. |
+| Domein          | Zaakgericht werken                                                                                                                                                                                |
+| Provider        | [Zaaktypecataloguscomponent](https://www.gemmaonline.nl/index.php/Omgevingswet/1.5/id-3ef9cdd9-631c-4d3e-88c3-f756423d6314) (GEMMA2)                                                              |
+| Consumer        | Componenten waarmee zaken behandeld worden (Zaakafhandelcomponenten; ZAC’s) en (andere) componenten die zaaktypegegevens raadplegen.                                                              |
+| Informatiemodel | [IMZTC, versie 2.1](https://www.gemmaonline.nl/index.php/Informatiemodel_Zaaktypen_(ImZTC)) (in-ontwikkeling)                                         |
+| Specificaties   | <https://ref.tst.vng.cloud/zrc/api/v1/schema>                                                                                                                                                     |
+| Bijzonderheden  |  Deze API omvat (vooralsnog) de resources Catalogussen, Zaaktypen, Informatieobjecttypen, Roltypen, Statustypen en Zaakobjecten                                                    |
+
+Deze API omvat (vooralsnog) de resources Zaken, Klantcontacten, Rollen, Statussen en Eigenschappen.
+
+Op dit moment biedt deze API alleen nog de mogelijkheid de zaaktypecatalogus te raadplegen middels GET.
+
+### Resource: catalogus
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een lijst van alle beschikbare CATALOGUSsen of een specifieke catalogus opvragen.         |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
```

Comment:
Ik zou bij 'Toevoegen', omdat deze operatie nog niet ontwikkeld is, zeggen '(operatie nog niet beschikbaar)'.  En dan bij voorwaarde en gevolg 'n.t.b.' ipv. 'n.v.t.'. Ze zijn wel degelijk van toepassing maar we kennen ze nog niet. In de andere specificaties hieronder naar analogie.

Reply:
Wat moet er dan nog toegevoegd worden? De operaties `catalogus_list` en `catalogus_read` zijn er al.

Reply:
Post! (spreek uit: poost).

Reply:
tekst aangepast.